tests/unit/drivers/image_generation/test_azure_openai_dalle_image_generation_driver.py

```diff
@@ -16,23 +14,23 @@ def driver(self):
             image_size="512x512",
         )
 
-    def test_init(self, driver):
+    def test_init(self, driver: AzureOpenAiDalleImageGenerationDriver):
         assert driver
 
     def test_init_requires_endpoint(self):
         with pytest.raises(TypeError):
             AzureOpenAiDalleImageGenerationDriver(
                 model="dall-e-3", client=Mock(), azure_deployment="dalle-deployment", image_size="512x512"
-            )
+            )  # type: ignore
```

Comment:
Why the ignore?

Reply:
We're intentionally failing to provide some fields to test validation, but pyright doesn't know that.